apis/validation.go

```diff
@@ -23,13 +24,20 @@ func ValidateInit(config *InitConfiguration) []error {
 	} else {
 		// Pod subnet was set through MasterConfiguration.Networking.PodSubnet
 		if config.MasterConfiguration.Networking.PodSubnet != config.Networking.PodSubnet {
-			errorList = append(errorList, fmt.Errorf("Configuration conflict: Networking.PodSubnet=%q, MasterConfiguration.Networking.PodSubnet=%q. Values should be identical, or MasterConfiguration.Networking.PodSubnet omitted.",
+			errorList = append(errorList, fmt.Errorf("configuration conflict: Networking.PodSubnet=%q, MasterConfiguration.Networking.PodSubnet=%q. Values should be identical, or MasterConfiguration.Networking.PodSubnet omitted",
 				config.Networking.PodSubnet, config.MasterConfiguration.Networking.PodSubnet))
 		}
 	}
 	if config.MasterConfiguration.Networking.DNSDomain != config.Networking.DNSDomain {
 		errorList = append(errorList, fmt.Errorf("configuration conflict: Networking.DNSDomain=%q, MasterConfiguration.Networking.DNSDomain=%q. Values should be identical, or MasterConfiguration.Networking.DNSDomain omitted",
 			config.Networking.DNSDomain, config.MasterConfiguration.Networking.DNSDomain))
 	}
+	if config.VIPConfiguration.RouterID != -1 {
```

Comment:
Hi Ojas, just curious why not look for NetworkInterface itself here, i think we have a separate check where we verify that both RouterID and NetworkInterface are provided.

Reply:
The idea was to only error here if this cluster is intended to be HA. If interface is empty, it could be done intentionally for a non-HA cluster.